plugins/ext_test/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## 1.0.0 (2020-03-09)
```

Comment:
You might want to bump the version of this plugin since it looks like you are changing to versioning based on setuptools_scm

Reply:
Something about the tag filtering falls apart on the CI servers. I don't know what the problem is. Options are either to switch to hard-coding the version or just keeping the version in lock-step with cmd2 proper.

Reply:
I feel strongly that we don't want to 100% tie the release cycle in precision lockstep, so it would be preferable to hard-code the versions of the plugins. 

You could do something like make a single plugin version in a file if you want and have the version read from there when building.

Reply:
I think I'm going to hard-code the setup.py to 0.2.0 and put in an issue to figure out why the tag filtering fails on the CI system. It may be possible to switch out how it does the tag filtering depending on whether we're running it on our system vs the CI system. We only really need the tag filtering to work when we invoke the task to publish to pypi and tag it. Doesn't matter in the context of the CI reporting.